modules/openideal_login/openideal_login.module

```diff
@@ -0,0 +1,194 @@
+use Drupal\Core\Link;
+use Drupal\Core\Routing\RouteMatch;
+use Drupal\Core\Url;
+use Drupal\openideal_user\Event\OpenidealUserEvents;
+use Drupal\openideal_user\Event\OpenidealUserJoinedSiteEvent;
+use Drupal\user\UserInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function openideal_login_form_user_register_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $form['title'] = [
+    '#type' => 'html_tag',
+    '#tag' => 'h1',
+    '#value' => t('Create your @site account', ['@site' => \Drupal::config('system.site')->get('name')]),
+    '#weight' => -1,
+  ];
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function openideal_login_form_user_login_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  /** @var \Drupal\social_api\Plugin\NetworkManager $network_manager */
+  $network_manager = \Drupal::service('plugin.network.manager');
+  $social_plugins = \Drupal::config('social_auth.settings')->get('auth');
+  $form['#attributes']['class'][] = 'user-login-form';
+  $form['container'] = [
+    '#type' => 'container',
+    '#attributes' => ['class' => ['user-login-form--container']],
+    'left' => [
+      '#type' => 'container',
+      '#attributes' => ['class' => ['user-login-form--container--left']],
+    ],
+    'right' => [
+      '#type' => 'container',
+      '#attributes' => ['class' => ['user-login-form--container--right']],
+    ],
+  ];
+
+  $entity_manager = \Drupal::entityTypeManager();
+  $block_entity = $entity_manager->getStorage('block_content')->loadByProperties(['info' => 'login']);
+  if (!empty($block_entity)) {
+    $block = $entity_manager->getViewBuilder('block_content')->view(current($block_entity), 'full');
+    $form['container']['right']['section']['block'] = $block;
+  }
+
+  $form['container']['left']['title'] = [
+    '#type' => 'html_tag',
+    '#tag' => 'h1',
+    '#attributes' => ['class' => ['user-login-form--container--title']],
+    '#value' => t('Login to @site', ['@site' => \Drupal::config('system.site')->get('name')]),
+  ];
+
+  foreach (['name', 'pass'] as $child) {
+    if (isset($form[$child])) {
+      unset($form[$child]['#description']);
+
+      if ($child == 'name') {
+        $form[$child]['#placeholder'] = t('Email Address');
+        $form[$child]['#weight'] = 3;
+      }
+      if ($child == 'pass') {
+        $form[$child]['#placeholder'] = t('Password');
+        $form[$child]['#weight'] = 4;
+      }
+
+      $form['container']['left'][$child] = $form[$child];
+      unset($form[$child]);
+    }
+  }
+
+  // Remove socials that aren't configured properly.
+  foreach ($social_plugins as $plugin_id => &$data) {
+    if (!$network_manager->createInstance($plugin_id)->getSdk()) {
+      unset($social_plugins[$plugin_id]);
+    }
+
+    $theme_path = \Drupal::service('theme_handler')->getTheme('openideal_theme')->getPath();
+    $prefix = 'Sign in with';
```

Comment:
Should be wrapped with t().

Reply:
The variable was unneeded, I removed it and put the 'Sign in with' inside t() function.